Jenkinsfile_nightly

```diff
@@ -0,0 +1,49 @@
+#!groovy
+
+properties([
+  // H allow predefined but random minute see https://en.wikipedia.org/wiki/Cron#Non-standard_characters
+  pipelineTriggers([cron('H 05 * * *')])
+])
+
+@Library("Infrastructure")
+
+def type = "java"
+def product = "ia"
+def component = "homeoffice-integration-api"
+
+def secrets = [
+  'ia-${env}': [
+
+    secret('idam-client-id', 'IA_IDAM_CLIENT_ID'),
+    secret('idam-secret', 'IA_IDAM_SECRET'),
+    secret('idam-redirect-uri', 'IA_IDAM_REDIRECT_URI'),
+    secret('s2s-secret', 'IA_S2S_SECRET'),
+    secret('s2s-microservice', 'IA_S2S_MICROSERVICE'),
+    secret('ccd-url', 'CCD_URL'),
+    secret('ccd-gw-url', 'CCD_GW_URL'),
+    secret('dm-url', 'DM_URL'),
+    secret('idam-url', 'IDAM_URL'),
+    secret('s2s-url', 'S2S_URL'),
+    secret('prof-ref-data-url', 'PROF_REF_DATA_URL')
```

Comment:
We should only add what we need here.  IDAM and S2S is ok.  The rest we can add as we need them.

Reply:
Removed other variables, kept only IDAM and S2S